selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/TestWebElementRenderChecker.java

```diff
@@ -87,25 +87,30 @@ private Boolean dimensionsAreEquivalent(
   }
 
   private void waitElementIsStatic(FluentWait<WebDriver> webDriverWait, WebElement webElement) {
-    AtomicInteger beforeDimensionSumWithShift = new AtomicInteger(0);
+    AtomicInteger dimensionSum = new AtomicInteger(0);
```

Comment:
just `dimension`, please :-)
Sum - is just the name of operation which is used to calculate the value.

Reply:
Also, the default constructor set value to 0 by default.

Reply:
on my opinion, it will be not enough clear in the method context

Reply:
ok, I agree, if change the second variable too, it will be clear enough

Reply:
You are partly right. What about name `dimensionHash`?